app/views/layouts/application.html.erb

```diff
@@ -6,8 +6,8 @@
 
   <script>
     var dataLayer = [{
-      'dimension1': '<%= current_user.organisation_slug %>',
-      'cp_uid': '<%= current_user.uid %>'
+      'user-organisation': '<%= current_user.organisation_slug %>',
+      'cp-uid': '<%= current_user.uid %>'
```

Comment:
Yay, much nicer. So kebab case all the way for dataLayer attributes?

Reply:
Yup :-).